core/types.ml

```diff
@@ -3494,14 +3495,14 @@ let test_type_roundtrip : datatype (* -> tycon_environment *) -> string -> bool
    *    (\* TODO compare if the parsed type is isomorphic to t *\)
    *    true *)
   (* TODO dependency cycle; wanna use DesugarDatatypes.read or parts of it *)
-  true
+  failwith "Testing for type round trip is not implemented."
```

Comment:
Not here. You should define any testing logic in a separate module.

Reply:
I'll remove that entirely for now, maybe in the future it would be nice to have though.